.github/workflows/tests.yml

```
@@ -0,0 +1,74 @@
+        python -m pip install --upgrade pip
+        pip install poetry==1.7.1
+        poetry install
+    - name: Run linters
+      run: make lint-check
+
+    - name: Run unit-tests
+      run: make unit-tests
+
+# Set Node.js version to 18 for running Cypress and npm commands
+    - name: Set up Node.js
+      uses: actions/setup-node@v3
```

actionlint (1.7.7)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)